core/controllers/feedback.py

```diff
@@ -213,7 +218,38 @@ def get(self, exploration_id):
 
 
 class UnsentFeedbackEmailHandler(base.BaseHandler):
-    """Handler task of sending emails of feedback messages.
-    This is yet to be implemented."""
+    """Handler task of sending emails of feedback messages."""
+
     def post(self):
-        pass
+        payload = json.loads(self.request.body)
+        user_id = payload['user_id']
+        references = feedback_services.get_feedback_message_references(user_id)
+        transaction_services.run_in_transaction(
+            feedback_services.update_feedback_email_retries, user_id)
+
+        messages = {}
+        for reference in references:
+            message = feedback_services.get_message(
+                reference.exploration_id, reference.thread_id,
+                reference.message_id)
+
+            exploration = exp_services.get_exploration_by_id(
+                reference.exploration_id)
+
+            message_text = message.text
+            if len(message_text) > 200:
+                message_text = message_text[:200] + '...'
+
+            reference_dict = {
```

Comment:
It's a small thing, but I think that for clarity you ought to declare this inline in line 250, otherwise it looks like it's used more generally.